### ISSUE TITLE
[Bug] Google drive plugin outdated

### ISSUE THREAD

**Issue by khewweifeng** [jlesage/docker-jdownloader-2#182]
### Current Behavior

![image](https://github.com/jlesage/docker-jdownloader-2/assets/54496743/b7c8ac0d-7c2d-490b-a82a-22c47c47b519)

### Expected Behavior

Able to download with account manager setup - https://support.jdownloader.org/Knowledgebase/Article/View/account-cookie-login-instructions

### Steps To Reproduce

1. Followed this guide (option 1) https://support.jdownloader.org/Knowledgebase/Article/View/account-cookie-login-instructions
2. Prompted plugin outdated (previously and yesterday still working) 

### Environment

- OS:
- OS version:
- CPU:
- Docker version:
- Device model:
- Browser/OS: 

![image](https://github.com/jlesage/docker-jdownloader-2/assets/54496743/e10596c3-f99f-441b-af1d-c39e9aa386f6)

root@raspberrypi:/home/pi# docker version
Client: Docker Engine - Community
 Version:           24.0.7
 API version:       1.43
 Go version:        go1.20.10
 Git commit:        afdd53b
 Built:             Thu Oct 26 09:08:29 2023
 OS/Arch:           linux/arm64
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          24.0.7
  API version:      1.43 (minimum version 1.12)
  Go version:       go1.20.10
  Git commit:       311b9ff
  Built:            Thu Oct 26 09:08:29 2023
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          1.6.25
  GitCommit:        d8f198a4ed8892c764191ef7b3b06d8a2eeb5c7f
 runc:
  Version:          1.1.10
  GitCommit:        v1.1.10-0-g18a0cb0
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0

### Container creation

---
# https://github.com/jlesage/docker-jdownloader-2
services:
  jdownloader:
    image: jlesage/jdownloader-2:latest
    container_name: jdownloader
    environment:
      - USER_ID=1000
      - GROUP_ID=100
      - TZ=Asia/Kuala_Lumpur
    volumes:
      - /srv/dev-disk-by-uuid-e69bdfc6-9d0f-4f54-a780-a400120d4859/samsung-old-hdd/jdownloader/config:/config
      - /srv/dev-disk-by-uuid-e69bdfc6-9d0f-4f54-a780-a400120d4859/samsung-old-hdd/jdownloader/output:/output
    ports:
      - 5800:5800
    restart: unless-stopped

### Container log

```text
[init        ] container is starting...
[cont-env    ] loading container environment variables...
[cont-env    ] APP_NAME: loading...
[cont-env    ] DISPLAY: executing...
[cont-env    ] DISPLAY: terminated successfully.
[cont-env    ] DISPLAY: loading...
[cont-env    ] DOCKER_IMAGE_PLATFORM: loading...
[cont-env    ] DOCKER_IMAGE_VERSION: loading...
[cont-env    ] GTK2_RC_FILES: executing...
[cont-env    ] GTK2_RC_FILES: terminated successfully.
[cont-env    ] GTK2_RC_FILES: loading...
[cont-env    ] GTK_THEME: executing...
[cont-env    ] GTK_THEME: terminated successfully.
[cont-env    ] GTK_THEME: loading...
[cont-env    ] HOME: loading...
[cont-env    ] INSTALL_PACKAGES_INTERNAL: executing...
[cont-env    ] INSTALL_PACKAGES_INTERNAL: terminated successfully.
[cont-env    ] INSTALL_PACKAGES_INTERNAL: loading...
[cont-env    ] QT_STYLE_OVERRIDE: executing...
[cont-env    ] QT_STYLE_OVERRIDE: terminated successfully.
[cont-env    ] QT_STYLE_OVERRIDE: loading...
[cont-env    ] TAKE_CONFIG_OWNERSHIP: loading...
[cont-env    ] XDG_CACHE_HOME: loading...
[cont-env    ] XDG_CONFIG_HOME: loading...
[cont-env    ] XDG_DATA_HOME: loading...
[cont-env    ] XDG_RUNTIME_DIR: loading...
[cont-env    ] XDG_STATE_HOME: loading...
[cont-env    ] container environment variables initialized.
[cont-secrets] loading container secrets...
[cont-secrets] container secrets loaded.
[cont-init   ] executing container initialization scripts...
[cont-init   ] 10-certs.sh: executing...
[cont-init   ] 10-certs.sh: terminated successfully.
[cont-init   ] 10-check-app-niceness.sh: executing...
[cont-init   ] 10-check-app-niceness.sh: terminated successfully.
[cont-init   ] 10-clean-logmonitor-states.sh: executing...
[cont-init   ] 10-clean-logmonitor-states.sh: terminated successfully.
[cont-init   ] 10-clean-tmp-dir.sh: executing...
[cont-init   ] 10-clean-tmp-dir.sh: terminated successfully.
[cont-init   ] 10-fontconfig-cache-dir.sh: executing...
[cont-init   ] 10-fontconfig-cache-dir.sh: terminated successfully.
[cont-init   ] 10-init-users.sh: executing...
[cont-init   ] 10-init-users.sh: terminated successfully.
[cont-init   ] 10-nginx.sh: executing...
[cont-init   ] 10-nginx.sh: terminated successfully.
[cont-init   ] 10-openbox.sh: executing...
[cont-init   ] 10-openbox.sh: terminated successfully.
[cont-init   ] 10-pkgs-mirror.sh: executing...
[cont-init   ] 10-pkgs-mirror.sh: terminated successfully.
[cont-init   ] 10-set-tmp-dir-perms.sh: executing...
[cont-init   ] 10-set-tmp-dir-perms.sh: terminated successfully.
[cont-init   ] 10-vnc-password.sh: executing...
[cont-init   ] 10-vnc-password.sh: terminated successfully.
[cont-init   ] 10-web-data.sh: executing...
[cont-init   ] 10-web-data.sh: terminated successfully.
[cont-init   ] 10-x11-unix.sh: executing...
[cont-init   ] 10-x11-unix.sh: terminated successfully.
[cont-init   ] 10-xdg-runtime-dir.sh: executing...
[cont-init   ] 10-xdg-runtime-dir.sh: terminated successfully.
[cont-init   ] 15-cjk-font.sh: executing...
[cont-init   ] 15-cjk-font.sh: terminated successfully.
[cont-init   ] 15-install-pkgs.sh: executing...
[cont-init   ] 15-install-pkgs.sh: terminated successfully.
[cont-init   ] 55-jdownloader2.sh: executing...
[cont-init   ] 55-jdownloader2.sh: terminated successfully.
[cont-init   ] 85-take-config-ownership.sh: executing...
[cont-init   ] 85-take-config-ownership.sh: terminated successfully.
[cont-init   ] 89-info.sh: executing...
    ╭――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――╮
    │                                                                      │
    │ Application:           JDownloader 2                                 │
    │ Application Version:   n/a                                           │
    │ Docker Image Version:  23.11.2                                       │
    │ Docker Image Platform: linux/arm64                                   │
    │                                                                      │
    ╰――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――╯
[cont-init   ] 89-info.sh: terminated successfully.
[cont-init   ] all container initialization scripts executed.
[init        ] giving control to process supervisor.
[supervisor  ] loading services...
[supervisor  ] loading service 'default'...
[supervisor  ] loading service 'app'...
[supervisor  ] loading service 'gui'...
[supervisor  ] loading service 'xvnc'...
[supervisor  ] loading service 'openbox'...
[supervisor  ] loading service 'nginx'...
[supervisor  ] loading service 'certsmonitor'...
[supervisor  ] service 'certsmonitor' is disabled.
[supervisor  ] loading service 'logrotate'...
[supervisor  ] loading service 'logmonitor'...
[supervisor  ] service 'logmonitor' is disabled.
[supervisor  ] all services loaded.
[supervisor  ] starting services...
[supervisor  ] starting service 'xvnc'...
[xvnc        ] Xvnc TigerVNC 1.13.1 - built Nov 17 2023 22:53:35
[xvnc        ] Copyright (C) 1999-2022 TigerVNC Team and many others (see README.rst)
[xvnc        ] See https://www.tigervnc.org for information on TigerVNC.
[xvnc        ] Underlying X server release 12014000
[xvnc        ] Sat Nov 25 22:42:16 2023
[xvnc        ]  vncext:      VNC extension running!
[xvnc        ]  vncext:      Listening for VNC connections on /tmp/vnc.sock (mode 0660)
[xvnc        ]  vncext:      Listening for VNC connections on all interface(s), port 5900
[xvnc        ]  vncext:      created VNC server for screen 0
[supervisor  ] starting service 'openbox'...
[supervisor  ] starting service 'nginx'...
[nginx       ] Listening for HTTP connections on port 5800.
[supervisor  ] starting service 'app'...
[supervisor  ] all services started.
[xvnc        ] Sat Nov 25 22:43:37 2023
[xvnc        ]  Connections: accepted: /tmp/vnc.sock
[xvnc        ]  SConnection: Client needs protocol version 3.8
[xvnc        ]  SConnection: Client requests security type None(1)
[xvnc        ]  VNCSConnST:  Server default pixel format depth 24 (32bpp) little-endian rgb888
[xvnc        ]  VNCSConnST:  Client pixel format depth 24 (32bpp) little-endian bgr888
[xvnc        ] Sat Nov 25 22:43:45 2023
[xvnc        ]  VNCSConnST:  closing /tmp/vnc.sock: Clean disconnection
[xvnc        ]  EncodeManager: Framebuffer updates: 3
[xvnc        ]  EncodeManager:   Tight:
[xvnc        ]  EncodeManager:     Solid: 5 rects, 2.0736 Mpixels
[xvnc        ]  EncodeManager:            80 B (1:103681 ratio)
[xvnc        ]  EncodeManager:     Bitmap RLE: 1 rects, 160 pixels
[xvnc        ]  EncodeManager:                 68 B (1:9.58824 ratio)
[xvnc        ]  EncodeManager:   Total: 6 rects, 2.07376 Mpixels
[xvnc        ]  EncodeManager:          148 B (1:56048.1 ratio)
[xvnc        ]  Connections: closed: /tmp/vnc.sock
[xvnc        ]  ComparingUpdateTracker: 0 pixels in / 0 pixels out
[xvnc        ]  ComparingUpdateTracker: (1:nan ratio)
[xvnc        ]  Connections: accepted: /tmp/vnc.sock
[xvnc        ]  SConnection: Client needs protocol version 3.8
[xvnc        ]  SConnection: Client requests security type None(1)
[xvnc        ]  VNCSConnST:  Server default pixel format depth 24 (32bpp) little-endian rgb888
[xvnc        ]  VNCSConnST:  Client pixel format depth 24 (32bpp) little-endian bgr888
[xvnc        ] Sat Nov 25 22:44:45 2023
[xvnc        ]  ComparingUpdateTracker: 91.0977 Mpixels in / 6.2442 Mpixels out
[xvnc        ]  ComparingUpdateTracker: (1:14.5892 ratio)
[xvnc        ] Sat Nov 25 22:46:27 2023
[xvnc        ]  ComparingUpdateTracker: 57.4111 Mpixels in / 5.39944 Mpixels out
[xvnc        ]  ComparingUpdateTracker: (1:10.6328 ratio)
[xvnc        ] Sat Nov 25 22:47:24 2023
[xvnc        ]  ComparingUpdateTracker: 69.9122 Mpixels in / 13.2542 Mpixels out
[xvnc        ]  ComparingUpdateTracker: (1:5.27472 ratio)
[xvnc        ] Sat Nov 25 22:47:30 2023
[xvnc        ]  ComparingUpdateTracker: 4.68933 Mpixels in / 2.01612 Mpixels out
[xvnc        ]  ComparingUpdateTracker: (1:2.32592 ratio)
[xvnc        ] Sat Nov 25 22:47:56 2023
[xvnc        ]  ComparingUpdateTracker: 20.4829 Mpixels in / 4.9151 Mpixels out
[xvnc        ]  ComparingUpdateTracker: (1:4.16735 ratio)
[xvnc        ] Sat Nov 25 23:10:34 2023
[xvnc        ]  VNCSConnST:  closing /tmp/vnc.sock: Clean disconnection
[xvnc        ]  EncodeManager: Framebuffer updates: 8850
[xvnc        ]  EncodeManager:   Tight:
[xvnc        ]  EncodeManager:     Solid: 6.474 krects, 41.9328 Mpixels
[xvnc        ]  EncodeManager:            101.156 KiB (1:1620.03 ratio)
[xvnc        ]  EncodeManager:     Bitmap RLE: 4.343 krects, 907.806 kpixels
[xvnc        ]  EncodeManager:                 129.711 KiB (1:27.731 ratio)
[xvnc        ]  EncodeManager:     Indexed RLE: 36.679 krects, 19.2021 Mpixels
[xvnc        ]  EncodeManager:                  4.51196 MiB (1:16.3277 ratio)
[xvnc        ]  EncodeManager:   Tight (JPEG):
[xvnc        ]  EncodeManager:     Full Colour: 13.083 krects, 157.374 Mpixels
[xvnc        ]  EncodeManager:                  89.849 MiB (1:6.68327 ratio)
[xvnc        ]  EncodeManager:   Total: 60.579 krects, 219.417 Mpixels
[xvnc        ]  EncodeManager:          94.5864 MiB (1:8.85648 ratio)
[xvnc        ]  Connections: closed: /tmp/vnc.sock
[xvnc        ]  ComparingUpdateTracker: 1.21809 Gpixels in / 138.529 Mpixels out
[xvnc        ]  ComparingUpdateTracker: (1:8.79305 ratio)
[xvnc        ] Sat Nov 25 23:40:31 2023
[xvnc        ]  Connections: accepted: /tmp/vnc.sock
[xvnc        ]  SConnection: Client needs protocol version 3.8
[xvnc        ]  SConnection: Client requests security type None(1)
[xvnc        ]  VNCSConnST:  Server default pixel format depth 24 (32bpp) little-endian rgb888
[xvnc        ]  VNCSConnST:  Client pixel format depth 24 (32bpp) little-endian bgr888
[xvnc        ] Sat Nov 25 23:42:13 2023
[xvnc        ]  VNCSConnST:  closing /tmp/vnc.sock: Clean disconnection
[xvnc        ]  EncodeManager: Framebuffer updates: 1634
[xvnc        ]  EncodeManager:   Tight:
[xvnc        ]  EncodeManager:     Solid: 1.202 krects, 7.81037 Mpixels
[xvnc        ]  EncodeManager:            18.7812 KiB (1:1625.2 ratio)
[xvnc        ]  EncodeManager:     Bitmap RLE: 1.02 krects, 709.203 kpixels
[xvnc        ]  EncodeManager:                 32.2129 KiB (1:86.3716 ratio)
[xvnc        ]  EncodeManager:     Indexed RLE: 4.115 krects, 3.79661 Mpixels
[xvnc        ]  EncodeManager:                  775.625 KiB (1:19.1829 ratio)
[xvnc        ]  EncodeManager:   Tight (JPEG):
[xvnc        ]  EncodeManager:     Full Colour: 4.539 krects, 25.8826 Mpixels
[xvnc        ]  EncodeManager:                  19.5731 MiB (1:5.04705 ratio)
[xvnc        ]  EncodeManager:   Total: 10.876 krects, 38.1988 Mpixels
[xvnc        ]  EncodeManager:          20.3803 MiB (1:7.15598 ratio)
[xvnc        ]  Connections: closed: /tmp/vnc.sock
[xvnc        ]  ComparingUpdateTracker: 212.822 Mpixels in / 28.0042 Mpixels out
[xvnc        ]  ComparingUpdateTracker: (1:7.59965 ratio)
[xvnc        ] Sat Nov 25 23:42:50 2023
[xvnc        ]  Connections: accepted: /tmp/vnc.sock
[xvnc        ]  SConnection: Client needs protocol version 3.8
[xvnc        ]  SConnection: Client requests security type None(1)
[xvnc        ] Sat Nov 25 23:42:51 2023
[xvnc        ]  VNCSConnST:  Server default pixel format depth 24 (32bpp) little-endian rgb888
[xvnc        ]  VNCSConnST:  Client pixel format depth 24 (32bpp) little-endian bgr888
[xvnc        ] Sat Nov 25 23:48:47 2023
[xvnc        ]  VNCSConnST:  closing /tmp/vnc.sock: Clean disconnection
[xvnc        ]  EncodeManager: Framebuffer updates: 2322
[xvnc        ]  EncodeManager:   Tight:
[xvnc        ]  EncodeManager:     Solid: 1.114 krects, 5.23539 Mpixels
[xvnc        ]  EncodeManager:            17.4062 KiB (1:1175.66 ratio)
[xvnc        ]  EncodeManager:     Bitmap RLE: 872 rects, 528.599 kpixels
[xvnc        ]  EncodeManager:                 30.624 KiB (1:67.7592 ratio)
[xvnc        ]  EncodeManager:     Indexed RLE: 10.039 krects, 6.35817 Mpixels
[xvnc        ]  EncodeManager:                  1.35268 MiB (1:18.0157 ratio)
[xvnc        ]  EncodeManager:   Tight (JPEG):
[xvnc        ]  EncodeManager:     Full Colour: 2.726 krects, 21.0958 Mpixels
[xvnc        ]  EncodeManager:                  15.7775 MiB (1:5.10255 ratio)
[xvnc        ]  EncodeManager:   Total: 14.751 krects, 33.218 Mpixels
[xvnc        ]  EncodeManager:          17.177 MiB (1:7.38691 ratio)
[xvnc        ]  Connections: closed: /tmp/vnc.sock
[xvnc        ]  ComparingUpdateTracker: 236.277 Mpixels in / 22.1029 Mpixels out
[xvnc        ]  ComparingUpdateTracker: (1:10.6898 ratio)
[xvnc        ] Sun Nov 26 10:54:39 2023
[xvnc        ]  Connections: accepted: /tmp/vnc.sock
[xvnc        ]  SConnection: Client needs protocol version 3.8
[xvnc        ]  SConnection: Client requests security type None(1)
[xvnc        ]  VNCSConnST:  Server default pixel format depth 24 (32bpp) little-endian rgb888
[xvnc        ]  VNCSConnST:  Client pixel format depth 24 (32bpp) little-endian bgr888
[xvnc        ] Sun Nov 26 10:55:24 2023
[xvnc        ]  VNCSConnST:  closing /tmp/vnc.sock: Clean disconnection
[xvnc        ]  EncodeManager: Framebuffer updates: 348
[xvnc        ]  EncodeManager:   Tight:
[xvnc        ]  EncodeManager:     Solid: 707 rects, 3.762 Mpixels
[xvnc        ]  EncodeManager:            11.0469 KiB (1:1331.02 ratio)
[xvnc        ]  EncodeManager:     Bitmap RLE: 20 rects, 16.535 kpixels
[xvnc        ]  EncodeManager:                 642 B (1:103.396 ratio)
[xvnc        ]  EncodeManager:     Indexed RLE: 432 rects, 476.146 kpixels
[xvnc        ]  EncodeManager:                  75.2725 KiB (1:24.7768 ratio)
[xvnc        ]  EncodeManager:   Tight (JPEG):
[xvnc        ]  EncodeManager:     Full Colour: 1.388 krects, 17.3237 Mpixels
[xvnc        ]  EncodeManager:                  6.99413 MiB (1:9.45087 ratio)
[xvnc        ]  EncodeManager:   Total: 2.547 krects, 21.5784 Mpixels
[xvnc        ]  EncodeManager:          7.07904 MiB (1:11.6321 ratio)
[xvnc        ]  Connections: closed: /tmp/vnc.sock
[xvnc        ]  ComparingUpdateTracker: 54.1929 Mpixels in / 18.1441 Mpixels out
[xvnc        ]  ComparingUpdateTracker: (1:2.98681 ratio)
[xvnc        ] Sun Nov 26 10:57:50 2023
[xvnc        ]  Connections: accepted: /tmp/vnc.sock
[xvnc        ]  SConnection: Client needs protocol version 3.8
[xvnc        ] Sun Nov 26 10:57:51 2023
[xvnc        ]  SConnection: Client requests security type None(1)
[xvnc        ]  VNCSConnST:  Server default pixel format depth 24 (32bpp) little-endian rgb888
[xvnc        ]  VNCSConnST:  Client pixel format depth 24 (32bpp) little-endian bgr888

END OF LINE
█
```

**Comment by jlesage** [comment]
Is your JDownloader up-to-date? You can force update check from the Help->Find Updates menu.

**Comment by khewweifeng** [comment]
Yes, it show latest version already.  Let me try again and see able to reproduce or not.
________________________________

**Comment by khewweifeng** [comment]
It seems ok already after I remove and readd the account
